### ISSUE TITLE
sailsjs memory usage & other performance profiles?

### ISSUE THREAD

**Issue by ItsLeeOwen** [balderdashy/sails#2743]
I'm previewing sailsjs v0.11.0, running the default app you get with "sails new".  While running the app is sitting idle at 60-70MB memory usage.  That seems high to me, and I'm assuming it's a result of the automagic inside of sailsjs.  Similar baseline apps that I've setup w/ express4 tend to check in around 15-25mb in idle memory usage, which is also high when compared to a similar go web server.

Are there any performance profiles of sailsjs apps handling various simulated stress-levels?

**Comment by tjwebb** [comment]
> go web server.

Comparing any node.js application to a `go` application is comparing apples to oranges. I can write a web server in C++ that uses 1MB of ram.

If you have a large number of ORMs, you can save 10-15MB of RAM on the spot by disabling dynamic finders: https://github.com/balderdashy/waterline/pull/594#issuecomment-52437235.

**Comment by CWyrtzen** [comment]
@tjwebb I see your labels above, is there anything we can do to move this to a resolution/close?

**Comment by sailsbot** [comment]
Thanks for posting, @ItsLeeOwen.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
